src/lib/tabs/tab-body.html

```diff
@@ -1,4 +1,6 @@
 <div class="mat-tab-body-content" #content
+     (swipe)="onSwipe.emit($event)"
+     [style.touch-action]="'pan-y'"
```

Comment:
I spent a bit too much time tracking this down. `touch-action: none` is coming from our material `GestureConfig`; adding the swipe gesture always sets the touch action to `pan-y` + `pan-x` (equivalent to `none`). 

The "correct" fix for this would be to change how our gesture config add recognizers for certain events, but that's more complicated than we want to do right now. I filed #6484 to track this. Can you make your comment something like
```html
<!-- touch-action workaround for https://github.com/angular/material2/issues/6484 -->
```
Comment can go above the element (I don't think a comment inside of a tag is valid html). HTML comments are stripped when packaging the release